package-lock.json

```diff
@@ -10868,42 +10873,6 @@
         "which": "1.3.0"
       }
     },
-    "karma-coverage": {
-      "version": "github:douglasduteil/karma-coverage#a2c924a883af5ff1aa9edb2164c54b6a801cdcf5",
-      "dev": true,
-      "requires": {
-        "dateformat": "1.0.12",
-        "minimatch": "0.3.0"
-      },
-      "dependencies": {
-        "dateformat": {
-          "version": "1.0.12",
-          "resolved": "https://registry.npmjs.org/dateformat/-/dateformat-1.0.12.tgz",
-          "integrity": "sha1-nxJLZ1lMk3/3BpMuSmQsyo27/uk=",
-          "dev": true,
-          "requires": {
-            "get-stdin": "4.0.1",
-            "meow": "3.7.0"
-          }
-        },
-        "lru-cache": {
-          "version": "2.7.3",
-          "resolved": "https://registry.npmjs.org/lru-cache/-/lru-cache-2.7.3.tgz",
-          "integrity": "sha1-bUUk6LlV+V1PW1iFHOId1y+06VI=",
-          "dev": true
-        },
-        "minimatch": {
-          "version": "0.3.0",
-          "resolved": "https://registry.npmjs.org/minimatch/-/minimatch-0.3.0.tgz",
-          "integrity": "sha1-J12O2qxPG7MyZHIInnlJyDlGmd0=",
-          "dev": true,
-          "requires": {
-            "lru-cache": "2.7.3",
-            "sigmund": "1.0.1"
-          }
-        }
-      }
-    },
```

Comment:
One more quick question: why did this part come out? I checked the change logs but didn't see any note explaining changes to `karma-coverage`.